rollup/pipeline/pipeline.go

```diff
@@ -69,13 +69,16 @@ type Pipeline struct {
 
 func NewPipeline(
 	chain *core.BlockChain,
-	vmConfig *vm.Config,
+	vmConfig vm.Config,
 	state *state.StateDB,
 
 	header *types.Header,
 	nextL1MsgIndex uint64,
 	ccc *circuitcapacitychecker.CircuitCapacityChecker,
 ) *Pipeline {
+	// make sure we are not sharing a tracer with the caller and not in debug mode
+	vmConfig.Tracer = nil
+	vmConfig.Debug = false
```

Comment:
Was this also an issue in the original `worker`? Since we also just called `core.ApplyTransaction` with `*w.chain.GetVMConfig()`, without resetting these fields.

Reply:
Yeah, but original worker had less concurrency going on. So that is probably why this issue never surfaced before.